src/main/webapp/resources/js/admin-abonnement.js

```diff
@@ -143,4 +174,34 @@ const abonnementsFunctions = function () {
         abonnementTable.ajax.reload(null, false);
     });
 
+
+    $(".datatable tbody").on('click', '.btn-assign', function () {
+        selectAllUsers();
+    });
```

Comment:
Is it dublicate listener to upper? $(document.body).on('click', '.btn-assign' ...